.github/workflows/test.yml

```diff
@@ -44,28 +44,158 @@ jobs:
           path: ./
 
-      - name: "Donwload Grype v0.54.0"
+      - name: Download Grype v0.54.0
         uses: ./download-grype # anchore/scan-action/download-grype
         with:
           grype-version: v0.54.0
 
-      - name: "Check Grype version before scan-action"
+      - name: Check Grype version before scan-action
         run: grype version | egrep "^Version:.*0.54.0$"
 
-      - name: "Scan test image"
+      - name: Scan test image
         uses: ./
         with:
-          image: "alpine:latest"
+          image: alpine:latest
           grype-version: v0.54.0 # set the same version to test that current Grype binary wasn't overwritten by the latest version
           fail-build: false # to prevent fail due to vuln:s on test image
 
-      - name: "Check Grype version after scan-action"
+      - name: Check Grype version after scan-action
         run: grype version | egrep "^Version:.*0.54.0$"
+
+  test-image-input:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - uses: ./
+        id: scan
+        with:
+          image: alpine:latest
+          fail-build: false
+
+      - name: Validate file exists
+        run: ls '${{ steps.scan.outputs.sarif }}'
+
+  test-directory-input:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - uses: ./
+        id: scan
+        with:
+          path: tests/fixtures/npm-project
+          fail-build: false
+
+      - name: Validate file exists
+        run: ls '${{ steps.scan.outputs.sarif }}'
+
+  test-sbom-input:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - uses: ./
+        id: scan
+        with:
+          sbom: tests/fixtures/test_sbom.spdx.json
+          fail-build: false
+
+      - name: Validate file exists
+        run: ls '${{ steps.scan.outputs.sarif }}'
+
+  test-table-output:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - uses: ./
+        id: scan
+        with:
+          output-format: table
+          image: alpine:latest
+          fail-build: false
+
+  test-windows-image:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - uses: ./
+        id: scan
+        with:
+          image: alpine:latest
+          fail-build: false
+
+      - name: Validate file exists
+        run: ls '${{ steps.scan.outputs.sarif }}'
+
+  test-windows-path:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - uses: ./
+        id: scan
+        with:
+          path: tests/fixtures/npm-project
+          fail-build: false
+
+      - name: Validate file exists
+        run: ls '${{ steps.scan.outputs.sarif }}'
```

Comment:
Just want to double-check that this way of checking whether a file exists works on Windows - don't have a windows machine in front of me to check myself.

Specifically, can you make sure that `ls some-non-existent-file` on Windows exits non-zero?